rs3-full-boss-guides/magister.txt

```diff
@@ -1,178 +1,310 @@
-      },
-      {
-        "name": "__Minions__",
-        "value": "⬥ Only happens once the Magister has performed a sufficient number of special attacks beforehand. [Example](https://imgur.com/4rbODea.mp4)\n\u00a0\u00a0\u00a0\u00a0• Two Soul Devourer minions spawn using random combat styles.\n\u00a0\u00a0\u00a0\u00a0• They have 10,000 HP and remain until either they die or the fight ends.\n\u00a0\u00a0\u00a0\u00a0• It is recommended to kill them unless the fight is at least halfway over.\n\u200B"
-      }
-    ]
-  }
-}
-.embed:json
+The Magister can use a random special attack, every 3 auto attacks. Here we cover what specials he has access to and how to deal with them.
+
+**Note that most players will tend to ignore most mechanics and just simply DPS through them with <:soulsplit:615613924506599497> to heal back any lost health.**
+
 .
-> **__Protecting Your Familiar__**
-.tag:familiars
-The main recommended familiars are a Ripper Demon <:ripperpouch:703581275453128714>, a Nihil <:shadownihil:854475441946230855> <:bloodnihil:854475442169053194> <:smokenihil:854475442227642388>, or a Blood Reaver <:reaverpouch:839903693837959228> (high end melee only). All of these familiars have relatively low lifepoints, particularly the ancient summoning familiars, so some attention is needed in order to keep your familiar alive.
+__**Power Blast**__
+.img:https://i.imgur.com/S1fJmLr.gif
+⬥ **Magister hits his staff on the ground, doing a red lightning attack for high ranged damage (~2.5k).**
+⬥ **How to deal with it:**
+    • Pray Range <:DeflectRange:544195488317046812>
+    • Resonance <:res:535541258844635148>
+    • Disruption Shield <:disrupt:535614336207552523>
 
 .
-**__Tricks to Protect Your Familiar__**
-⬥ Try to kick the Unstable Mixture at the Magister if your familiar is about to die, as this will otherwise frequently hit your familiar for 2,000 damage.
-    • 5 Mixtures hitting your Blood Reaver <:reaverpouch:839903693837959228> will kill it, 6 Mixtures will kill your Ripper Demon <:ripperpouch:703581275453128714>, and 10 will kill your Nihil <:shadownihil:854475441946230855> <:bloodnihil:854475442169053194> <:smokenihil:854475442227642388>.
+__**Barge/Stand Still**__
+.img:https://i.imgur.com/ier7amn.gif
+⬥ Magister will do one of two attacks depending on where the player is standing.
+    • **Player in MD**: Magister says `Stand still!` and hits once, dealing heavy (~2.5k) melee damage, stunning the player and disabling protection prayers briefly.
+    • **Player not in MD**: Magister says `You dare fight me?`, *and after 1.8s* barges the player's location dealing heavy melee damage in a 5x5 AoE, and disabling protection prayers briefly.
```

Comment:
"barges the player, dealing heavy..." or "barges to the player's location, dealing heavy..."